x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/utils/helpers.test.tsx

```diff
@@ -1806,7 +1807,7 @@ describe('Exception helpers', () => {
           label: 'Agent status',
         },
         {
-          id: 'crowdstrike.event.DeviceId',
+          id: 'device.id',
```

Comment:
suggest that you replace with the use of the `const` we have that stores the agent type field name in order to make this test more resilient.